src/SeqCli/Cli/Commands/AppInstance/CreateCommand.cs

```diff
@@ -48,12 +49,16 @@ public CreateCommand(SeqConnectionFactory connectionFactory, SeqCliConfig config
                 _settings.Add(name, valueText ?? "");
             });
 
+        Options.Add(
+            "stream",
+            "Stream incoming events",
+            _ => _streamIncomingEvents = true
```

Comment:
Could the argument here be a signal expression? I.e. `--stream` (no argument) would stream all events, `--stream=signal-1,signal-2` would stream the intersection of these two signals.

IIRC there's a `SignalExpressionPart.Parse` method which should make this pretty smooth 🤞 ...